### PR TITLE
Create user controller

### PR DESCRIPTION
**Pull request by edkajimenez**
## Summary
Create  user controller
Create CRUD methods
Create method to add and delete a book to an user
Handle exceptions
## Trello card
https://trello.com/c/2Pd2C8fU/21-crear-user-controller-con-las-operaciones-crud-de-user

### CODE REVIEW

File: src/main/java/wolox/training/controllers/UserController.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import wolox.training.exceptions.UserIdMismatchException;
+import wolox.training.exceptions.UserNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.models.User;
+import wolox.training.repositories.BookRepository;
+import wolox.training.repositories.UserRepository;
+
+@RestController
+@RequestMapping("/api/users")
+@Controller
```
Comment (zeminlu) [inline src/main/java/wolox/training/controllers/UserController.java]: Remove this Annotation. It's either @Controller (view oriented controller) or @RestController (API oriented controller).

File: src/main/java/wolox/training/controllers/UserController.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+public class UserController {
+
+    @Autowired
+    private UserRepository userRepository;
+    @Autowired
```
Comment (zeminlu) [inline src/main/java/wolox/training/controllers/UserController.java:30]: Empty line between variable declarations.

**Review by zeminlu — CHANGES_REQUESTED** [review]
Nice work! Just 2 minor things :)

File: src/main/java/wolox/training/controllers/UserController.java (hunk 1 above)
Comment (zeminlu) [reply src/main/java/wolox/training/controllers/UserController.java]: You'll probably need to remove the greeting endpoint too, if it's still there.

Comment (edkajimenez) [reply src/main/java/wolox/training/controllers/UserController.java]: Oh! got it

**Review by zeminlu — APPROVED** [review]

**Pull request merged**